sql/core/src/main/scala/org/apache/spark/sql/execution/metric/SQLMetrics.scala

```diff
@@ -193,6 +194,14 @@ object SQLMetrics {
     }
   }
 
+  def stringStats(value: Long): String = {
```

Comment:
we should handle stats in `stringValue`, different metrics may need to look at different stats and display different things.

Reply:
one example: we can also display the difference between real row count and estimated row count, e.g. `10X`, `0.01X`, etc.